src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo List | Stanley</title>
+    <script src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/a1e854484d.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+    <script defer src="main.js"></script>
+  </head>
+  <body>
+    <main class="main-content">
+      <section class="to-do">
+        <div class="todo-container">
+          <ul class="task-list"></ul>
+          <div class="completed">
+            <button class="delete-tasks">Clear all completed</button>
+          </div>
```

Comment:
- [ ] Kindly make sure that the "Clear all completed" button matches the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/214345890-740fcf4d-13de-475f-968e-37db5a3881f9.png) | ![image](https://user-images.githubusercontent.com/62290545/214345966-08b4799a-ac2d-4bc6-9735-de548c5514f6.png) |